code/game/objects/items/devices/paicard.dm

```diff
@@ -92,10 +92,26 @@
 					to_chat(pai, "<span class='userdanger'>Your mental faculties leave you.</span>")
 					to_chat(pai, "<span class='rose'>oblivion... </span>")
 					qdel(pai)
-		if(href_list["wires"])
-			var/wire = text2num(href_list["wires"])
-			if(pai.radio)
-				pai.radio.wires.cut(wire)
+		if(href_list["toggle_transmit"])

```

Comment:
Could you compress this a bit please. I'm usually not one for code golfing but this is a lot of repetition.

Reply:
The original file tried to pass "wires" and "wire" for both transmit and receive and it never worked. I split the code out and copied the flavor from the holoprojector. I can remove the flavortext, but I'm not really sure how to keep myself from falling into the same pitfall that the person before me did.

Reply:
```DM
if(href_list["toggle_transmit"] || href_list["toggle_receive"])
    var/transmitting = href_list["toggle_transmit"] //it can't be both so if we know it's not transmitting it must be receiving. 
    var/message_handler_boolean
    if(transmitting)
        pai.can_transmit = !pai.can_transmit
        pai.radio.wires.cut(WIRE_TX) //should this really be happening regardless of disable/enable? 
    else //receiving
        pai.can_receive = !pai.can_receive
        pai.radio.wires.cut(WIRE_RX) //should this really be happening regardless of disable/enable?

    message_handler_boolean = (transmitting ? pai.can_transmit : pai.can_receive)
    to_chat(usr,"<span class='warning'>You [message_handler_boolean ? "enable" : "disable"] your pAI's [transmitting ? "outgoing" : "incoming"] radio transmissions!</span>")
    to_chat(pai,"<span class='warning'>Your owner has [message_handler_boolean ? "enabled" : "disabled"] your [transmitting ? "outgoing" : "incoming"] radio transmissions!</span>")
```
How about this 💪 ?

Reply:
Yes!